packages/admin-next/admin-sdk/src/lib/serve.ts

```diff
@@ -41,6 +42,8 @@ export async function serve(options: ServeOptions) {
     res.setHeader("Vary", "Origin, Cache-Control")
   }
 
+  router.use(compression())
```

Comment:
Should we pull the compression settings from the config (I see we have some there)?

Reply:
I think we might want to have a separate compression config in `projectConfig.admin` that we could pass here, as I don't necessarily think the compressions options you want generally for your server === the compression you want for the admin dashboard. But I think we can leave that for later, the default compression options seems to work great for this use case, but we should definetly open them up to people if they want to modify it for whatever reason.